Ghidra/Features/PDB/src/pdb/cpp/symbol.cpp

```diff
@@ -301,17 +310,17 @@ BSTR getBaseTypeName(IDiaSymbol * pSymbol) {
 }
 */
 
-IDiaSymbol * getType(IDiaSymbol * pSymbol) {
-	IDiaSymbol * pBaseType;
-	if (pSymbol->get_type( &pBaseType ) == 0) {
+CComPtr<IDiaSymbol> getType(IDiaSymbol& pSymbol) {
+    CComPtr<IDiaSymbol> pBaseType;
+	if (SUCCEEDED(pSymbol.get_type( &pBaseType ))) {
 		return pBaseType;
 	}
 	return NULL;
 }
-BSTR getTypeAsString(IDiaSymbol * pSymbol) {
-	BSTR typeStr ;
-	IDiaSymbol * pType;
-	if (pSymbol->get_type( &pType ) == 0) {
+std::wstring getTypeAsString(IDiaSymbol& pSymbol) {
+    std::wstring typeStr ;
```

Comment:
eliminate space before semicolon on line 321
Also... lines 321-322: each convert 4 spaces to 1 tab